frontend/class-frontend.php

```diff
@@ -1870,6 +1870,12 @@ class WPSEO_JSON_LD {
 	private $profiles = array();
 
 	/**
+	 * Holds the output array
+	 * @var array
+	 */
+	private $output = array();
```

Comment:
FYI: AFAIK those will not be picked up by phpDocumentor... now while this project might not use that, it is a good habit to document in a way that is compliant with phpDocumentor. Just my two pennies..